build.yaml

```diff
@@ -1 +1,8 @@
 projectType: extension
+additionalTestConfigs:
+  jdk11:
+    testJdkTool: TEMURIN-JDK11
+    mavenAdditionalArgs: -Dtest=none -DfailIfNoTests=false -DminVersion=4.3.0 -DruntimeProduct=MULE_EE
```

Comment:
I think you don't need the -DruntimeProduct=MULE_EE here

Reply:
I do, because the `<product>ALL</product>` was in the pom.xml and I decided to keep it after we talked. So I'm sending the flag to avoid modifying the connector original runtime configuration. Please let me know your thoughts on this.

Reply:
I don't understand why, if the connector must work in ALL products, the test with Java 11 states that it should only be tested with the EE version. Is this because of a known bug in the community version?

Reply:
I removed the runtimeProduct tag, because you are right, If java 8 runs with all, then java 11 should do that too.